python/grass/script/__init__.py

```diff
@@ -1,10 +1,222 @@
 """Python interface to launch GRASS GIS modules in scripts
 """
 
-from .core import *
-from .db import *
-from .raster import *
-from .raster3d import *
-from .vector import *
-from .utils import *
+from .core import (
+    call,
+    create_environment,
+    create_location,
+    create_project,
+    compare_key_value_text_files,
+    debug,
+    debug_level,
+    del_temp_region,
+    error,
+    exec_command,
+    fatal,
+    feed_command,
+    find_file,
+    find_program,
+    gisenv,
+    get_commands,
+    get_real_command,
+    handle_errors,
+    get_raise_on_error,
+    get_capture_stderr,
+    info,
+    legal_name,
+    list_grouped,
+    list_pairs,
+    list_strings,
+    locn_is_latlong,
+    make_command,
+    mapsets,
+    message,
+    overwrite,
+    parse_command,
+    parse_color,
+    parser,
+    percent,
+    pipe_command,
+    read_command,
+    region,
+    region_env,
+    run_command,
+    set_raise_on_error,
+    start_command,
+    sanitize_mapset_environment,
+    set_capture_stderr,
+    tempdir,
+    tempfile,
+    tempname,
+    use_temp_region,
+    version,
+    verbose,
+    verbosity,
+    warning,
+    write_command,
+    PIPE,
+)
+
+from .db import (
+    db_begin_transaction,
+    db_commit_transaction,
+    db_connection,
+    db_describe,
+    db_select,
+    db_table_exist,
+    db_table_in_vector,
+)
+
+from .raster import mapcalc, mapcalc_start, raster_history, raster_info, raster_what
+
+from .raster3d import mapcalc3d, raster3d_info
+
+from .utils import (
+    KeyValue,
+    basename,
+    decode,
+    encode,
+    float_or_dms,
+    parse_key_val,
+    try_remove,
+    diff_files,
+    try_rmdir,
+    text_to_string,
+    get_num_suffix,
+    split,
+    natural_sort,
+    naturally_sorted,
+    get_lib_path,
+    set_path,
+    clock,
+    legalize_vector_name,
+    append_node_pid,
+    append_uuid,
+    append_random,
+    separator,
+)
+
+from .vector import (
+    vector_db,
+    vector_layer_db,
+    vector_columns,
+    vector_history,
+    vector_info_topo,
+    vector_info,
+    vector_db_select,
+    vector_what,
+)
 from . import setup  # noqa: F401
+
+__all__ = [
+    "PIPE",
```

Comment:
The ordering looks like it mixes alphabetical ordering and order by the modules, I am not sure if there is any convention, I would prefer ordering by the modules, it's easier to check.

Reply:
I had initially ordered it by modules itself, however precommit reorders it every time i commit it to alphabetical style

Reply:
ok, then that's fine

Reply:
just remove the comments then, they are misleading